README.md

```diff
@@ -4,33 +4,39 @@
 The collective effort of the Project Pokémon (and other) communities to preserve as much event Pokémon information as possible.
 
 ### How do I use this?
-#### • Legality with [PKHeX](https://projectpokemon.org/home/files/file/1-pkhex/)
+#### • Legality with [PKHeX](https://projectpokemon.org/home/files/file/1-pkhex/) and [SysBot.NET](https://projectpokemon.org/home/files/file/4207-sysbotnet-automatic-build/)
 1. Download this repository.
-1. Extract the contents to a folder `mgdb` in the same folder as `PKHeX.exe`.
+1. Extract the contents to a folder `mgdb` in the same folder as `PKHeX.exe` or `SysBot.exe`.
 
-#### • Injection with PKHeX
+#### • PKM creation with PKHeX
 1. Find the specific wondercard or Pokémon file.
 1. Drag/drop the file onto PKHeX in the spot where you want that Pokémon.
 
-#### • Converting WC4 back to PCD
-1. Import/Export with PKHex
-2. Use the [PCDWC4 Converter](https://github.com/projectpokemon/PCDWC4Converter/releases/latest)
+#### • In-game Mystery Gift redemption
+* For Gen 3 events, use either the [WC3 Plugin](https://projectpokemon.org/home/files/file/4161-pkhex-plugin-wc3-plugin/) or the [Mystery Gift Tool](https://projectpokemon.org/home/forums/topic/39184-gen-iii-mystery-gift-tool-nintendo-events-wondercards-e-trainer-cards-and-e-berry-editor-and-more/)
+* For Gen 4, 5, 6 and 7 (3DS) events, use the [PKHeX Mystery Gift feature](https://projectpokemon.org/home/tutorials/save-editing/using-pkhex/importing-wondercards-r29/)
+* For Gen 7 (Switch), 8 and 9 events, use the [Switch Gift Data Manager](https://projectpokemon.org/home/forums/topic/62491-switch-gift-data-manager-import-wondercards-into-switch-games-by-faking-bcat-packages/)
 
-### How do I get Gen 8 Wondercards if they are not on the save file?
-Pokemon LGPE and Sword/Shield use a system called BCAT. BCAT data is synchronized when your game goes online.
-Use [JKSV](https://github.com/J-D-K/JKSV) to export the BCAT files. BCAT includes wondercards, wild area news, tournament rules, etc.
-See the section below on how to contribute your bcat export.
+#### • Import Raid blocks with PKHeX
+1. Follow our [tutorial](https://projectpokemon.org/tutorials/save-editing/gen-8_157/block-editing-r106/) to import the files into the corresponding blocks into your save file.
+1. Sword and Shield Raid blocks: `KDropRewards`, `KDaiEncount`, `KBonusRewards`, `KNormalEncount`, `KNormalEncountRigel1`, `KNormalEncountRigel2`. 
+1. Scarlet and Violet Raid blocks: `KBCATEventRaidIdentifier`, `KBCATFixedRewardItemArray`, `KBCATLotteryRewardItemArray`, `KBCATRaidEnemyArray`, `KBCATRaidPriorityArray`. 
```

Comment:
lgtm
My only other feedback is that it's not immediately obvious for someone who hasn't looked at pkhex's comments that the raid blocks for gen 9 are mapped as following
event_raid_identifier -> KBCATEventRaidIdentifier
fixed_reward_item_array -> KBCATFixedRewardItemArray
lottery_reward_item_array -> KBCATLotteryRewardItemArray
raid_enemy_array -> KBCATRaidEnemyArray
raid_priority_array -> KBCATRaidPriorityArray
otherwise looks good to me

Reply:
I considered to be more specific, but that would still not be noob friendly for users. Most user asks for 1:1 step guides.
Moreover, in my point of view, that's too much information to add in a general repository Readme. I prefer a clean Readme with external remindigs to other guides and links.
The idea was to make more detailed guides on the forums, like the topics/guides we have for Sword and Shield. That might take some times, but it'll be done.

Reply:
> I considered to be more specific, but that would still not be noob friendly for users. Most user asks for 1:1 step guides. Moreover, in my point of view, that's too much information to add in a general repository Readme. I prefer a clean Readme with external remindigs to other guides and links. The idea was to make more detailed guides on the forums, like the topics/guides we have for Sword and Shield. That might take some times, but it'll be done.

Awesome, all good

Reply:
> lgtm My only other feedback is that it's not immediately obvious for someone who hasn't looked at pkhex's comments that the raid blocks for gen 9 are mapped as following event_raid_identifier -> KBCATEventRaidIdentifier fixed_reward_item_array -> KBCATFixedRewardItemArray lottery_reward_item_array -> KBCATLotteryRewardItemArray raid_enemy_array -> KBCATRaidEnemyArray raid_priority_array -> KBCATRaidPriorityArray otherwise looks good to me

I can't find event_raid_identifier -> KBCATEventRaidIdentifier. 

How I can find it?

Reply:
That block has been labeled only recently in PKHeX. Use the Development build to find it. Otherwise you can look for `37B99B4D - 1295 Object 0x004` in the unlabeled blocks.

Reply:
Tysm!!